library/pid.inc

```diff
@@ -28,18 +28,17 @@ function setpid($new_pid)
 
   // Be careful not to clear the encounter unless the pid is really changing.
     if (!isset($_SESSION['pid']) || $pid != $new_pid_int || $pid != $_SESSION['pid']) {
-        $_SESSION['encounter'] = $encounter = 0;
+        $encounter = 0;
+        OpenEMR\Common\Session\SessionUtil::setSession($web_root, 'encounter', 0);
```

Comment:
Since these calls will be after globals.php (and autoloader), we can remove it from the `exclude-from-classmap`section in composer.json and include it like other classes (via `use ...` etc.).
Since this is also the case, we can also assume use `$GLOBALS['webroot']` within the function and not need a $web_root parameter.

Reply:
yep agree. going to bed though.:)

Reply:
Let me know if you want me to do this tonight (I may get carried away though and implement it in many places :) ). btw, the demographics.php scripts is screaming in speed :)

Reply:
Sure Brady, do what you want. I think we've have it now.